### PR TITLE
Added new timetable filters latestTime and offset. Also DateTime limitation explanation.

### PR DESCRIPTION
**Pull request by ivansams**

### CODE REVIEW

**Comment by bilo89za** [comment]
Looks good ;)

File: tapi/default.md [hunk 1]
```diff
@@ -773,14 +775,16 @@ A timetable of vehicles arriving and departing from a stop along their respectiv
...
 Retrieves a timetable for a stop, consisting of a list of occurrences of a vehicle calling at this stop in order of arrival time.
 
-`GET api/stops/{id}/timetables?earliestArrivalTime={DateTime}&limit={int}`
+`GET api/stops/{id}/timetables?earliestArrivalTime={DateTime}&latestArrivalTime={DateTime}limit={int}&offset={int}`
 
 | Parameter | Type | Description |
 | :-------------- | :--- | :---- |
 | id | [Identifier](#identifiers) | The identifier of the stop. |
-| earliestArrivalTime | [DateTime](#datetime) | The earliest arrival date and time to include in the timetable. |
+| earliestArrivalTime | [DateTime](#datetime) | The earliest arrival date and time to include in the timetable, inclusive. Defaults to Now.  |
```
Comment (davenewza) [inline tapi/default.md]: I would probably say "now" over "Now"

File: tapi/default.md [hunk 2]
```diff
@@ -961,16 +965,18 @@ A timetable of vehicles travelling on a line.
...
-`GET api/lines/{id}/timetables?earliestDepartureTime={DateTime}&departureStopId={stop}&arrivalStopId={stop}&limit={int}`
+`GET api/lines/{id}/timetables?earliestDepartureTime={DateTime}&latestDepartureTime={DateTime}&departureStopId={stop}&arrivalStopId={stop}&limit={int}&offset={int}`
 
 | Parameter | Type | Notes |
 | :-------------- | :--- | :---- |
 | id | [Identifier](#identifiers) | The identifier of the line. |
-| earliestDepartureTime | [DateTime](#datetime) | Optional earliest departure time on that line to be included in the timetable. |
-| departureStopId | [Identifier](#identifiers) | Optional stop identifier - bounds results to only occur after this stop. |
-| arrivalStopId | [Identifier](#identifiers) | Optional stop identifier - bounds results to only occur before this stop. |
+| earliestDepartureTime | [DateTime](#datetime) | Earliest departure time for the trip (departure time from the first stop on that line) to be included in the timetable. Defaults to Now. |
```
Comment (davenewza) [inline tapi/default.md]: "now"?

File: tapi/default.md [hunk 3]
```diff
@@ -289,6 +289,8 @@ ISO 8601 date and time strings can be represented as "2016-11-19T07:22Z" (7:22 A
 
 More information can be found [here](https://en.wikipedia.org/wiki/ISO_8601).
 
+Acceptable DateTime input values are limited to between 1 day in the past and 6 days in the future. This allows the user to query on any day of the week, but it should be remembered that public holidays, schedule changes and daylight savings time changes might mean that results for the same day might differ from week to week. Supplying a DateTime that exceeds this limitation will result in a 400 Bad Request response. In the case of a default that falls outside this limitation, the default will use the limitation value.
```
Comment (davenewza) [inline tapi/default.md]: "400 Bad Request" to be in bold and with the "status code" hyper link afterwards (see Accept type section for an example)

File: tapi/default.md (hunk 1 above)
Comment (davenewza) [inline tapi/default.md]: I wonder if it's worth add something like "See here for date range limitations."?

Comment (ivansams) [inline tapi/default.md]: I copied the casing from the journeys docs. I will update throughout the document with lower case.

Comment (ivansams) [inline tapi/default.md]: I considered it but I thought we already have the link in the table. I could add something like that in the description above?

Comment (davenewza) [inline tapi/default.md]: Ok cool. As long as we consistent I think.

File: tapi/default.md (hunk 2 above)
Comment (davenewza) [inline tapi/default.md]: Good point. Should we create a maintenance sticky for this?

**Pull request merged**